tests/StorageApiTestCase.php

```diff
@@ -766,7 +669,7 @@ protected function createDevBranchForTestCase(self $that)
         return $devBranch->createBranch($branchName);
     }
 
-    protected function getDefaultBranchId(self $that)
+    public function getDefaultBranchId(self $that)
```

Comment:
Tohle mě vůbec neuráží, nějak si nepamatuju, kde jsem to rozporoval... ale samozřejmě by to mohlo být v branch provideru :)